client/src/app/core/users/user.model.ts

```diff
@@ -149,4 +149,21 @@ export class User implements UserServerModel {
   updateAccountAvatar (newAccountAvatar: Avatar) {
     this.account.updateAvatar(newAccountAvatar)
   }
+
+  canUpload (): boolean {
```

Comment:
`isUploadDisabled` would be a better name, since this method does not check used quota (which may exceed the user quota)